geo-location.html

```diff
@@ -129,32 +139,47 @@
        */
 
       ready: function() {
+        this.fetch();
+      },
+
+      clear: function () {
+        this._setPosition(undefined);
```

Comment:
I think we should use `null` instead. `undefined` is a bit weird. It says, "geolocation doesn't have this property", when in fact it does. Can you init them to `null` in `properties`?